test/e2e/src/tests/test_ds_workflows.py

```diff
@@ -27,6 +27,7 @@ def test_ds_workflows(env_dict):
     api_instance = client.CoreV1Api()
 
     daemonsetPodLabelSelector = constants.AGENT_DAEMON_SET_PODS_LABEL_SELECTOR
+    ### dict get
```

Comment:
I dont see any change other this comment and if so, please revert both the test files.